templates/Layout/DNRoot_alerts.ss

```diff
@@ -13,18 +13,15 @@
 
 <div class="content">
 
-	<% if $AlertsConfigContent %>
-		<h3>Currently configured alerts</h3>
-		<pre>$AlertsConfigContent</pre>
+	<% if $AlertsConfigContent(master) %>
```

Comment:
confirmed, it's cached. and it's still a very quick check:

sprintf('git show --format=raw %s:.alerts.yml', $sha);